WireAPI/Tests/WireAPITests/Backend/PinnedKeyTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+
+@testable import WireAPI
+
+final class PinnedKeyTests: XCTestCase {
+
+    func testMatches() throws {
```

Comment:
Shouldn't we use the test naming convention here ? we can also omit the throws

Reply:
I will fix the `throws`. I can't think of a better name without modifying the test itself. Do you have any suggestions?

Reply:
The test is about asserting the sut matches or not some host, so it could be something like `testHostMatches` or something similar

Reply:
👍 I've renamed to `testMatchesHost` to better match the method name `matches(host: String)`.